service/src/test/java/bio/terra/workspace/amalgam/landingzone/azure/LandingZoneApiDispatchTest.java

```diff
@@ -120,21 +85,110 @@ public void listAzureLandingZoneResourcesByPurpose_Success_NoResults() throws Ex
   }
 
   @Test
-  public void listAzureLandingZoneResourcesByPurpose_ResourcePurpose_Success() throws Exception {
```

Comment:
Will restore the test:  listAzureLandingZoneResourcesByPurpose_ResourcePurpose_Success() - it is still valid.

Reply:
Also, I will have to merge in changes from TOAZ-221 first to fix the failing tests.